src/test/java/software/amazon/encryption/s3/S3EncryptionClientCompatibilityTest.java

```diff
@@ -210,6 +222,39 @@ public void AesGcmV2toV3WithInstructionFile() {
         deleteObject(BUCKET, objectKey, v3Client);
         v3Client.close();
     }
+    @Test
+    public void multipartPutObjectWithOptionsAndInstructionFileV2() throws IOException, InterruptedException, ExecutionException {
```

Comment:
This is useful for development, but we don't need to include it, since it's only testing V2 but not V3.